Sources/Purchasing/StoreKit1/ProductsFetcherSK1.swift

```diff
@@ -173,9 +173,7 @@ extension ProductsFetcherSK1: SKProductsRequestDelegate {
                     completion(.failure(error))
                 }
             } else {
-                let delayInSeconds = Int((self.requestTimeout.seconds / 10).rounded())
-
-                queue.asyncAfter(deadline: .now() + .seconds(delayInSeconds)) { [self] in
+                queue.asyncAfter(deadline: .now() + .seconds(3)) { [self] in
```

Comment:
Why 3?

Reply:
Human compiler optimization
`self.requestTimeout.seconds / 10`  -> `self.requestTimeout.seconds = 30` -> `3`

Reply:
But that assumes that the timeout is 30? 🤨

Reply:
Yup, check it out- https://github.com/RevenueCat/purchases-ios/pull/1556/files#diff-b080ecaed1fab9b6af95a6a40653aa15720509e51c3faa1aa71263f99a4ff1cbL30

Reply:
I'm confused. This class has a `requestTimeout` parameter` that can take any value. Before this delay was 1/10th of that value, but now it's hardcoded as 3 seconds, regardless of the value of that parameter. But what if we have a different `requestTimeout`?

Reply:
IMO, it would be better to have exponential backoff on the retry instead of just a percentage of the original value. I chose `3` purely because the default math works out to `3`. Stepping back for a second- I don't think there's a benefit to making the retry time be based on the request timeout value. It seems just as arbitrary as `3`.  Like, why does it matter that the retry is calculated based on a timeout? I could see retry based on network throughput/latency, but I don't think timeout value is a good proxy for that. 
To be clear, I'm not trying to be argumentative, I just think we could hardcode any value < 10 seconds here and still be fine 😄
Thoughts?

Reply:
> To be clear, I'm not trying to be argumentative,

Me neither, sorry! 😅 I was just confused cause I didn't understand how this is equivalent.

> Stepping back for a second- I don't think there's a benefit to making the retry time be based on the request timeout value

Well the main benefit is so that tests don't take 3 seconds when we try out the retry mechanism. That's why I changed this to just a few milliseconds.

> Like, why does it matter that the retry is calculated based on a timeout?

That's a good point, maybe we can have a separate parameter? One for timeout, another one for retry delays. Then the tests don't assume what this value is, and it can be very small.

However, even for production 3 seconds might be a lot. This could mean that a purchase adds 3 extra seconds?

Reply:
OOooooo ok ok, that makes sense. I'll add another param for retry delay, I like that.

Reply:
Actually... Imma leave this out and return it back to `let delayInSeconds = Int((self.requestTimeout.seconds / 10).rounded())`
Adding another parameter that is timeout/retry related really calls for a new enum in the configuration object and I don't think we need that yet.

Reply:
✅